tests/test_particle_track.py

```diff
@@ -163,14 +150,59 @@ def test_init_params():
 
 def test_previous_walk_data():
     # test of loading previously defined walk data
+    particle = particle_track.Particles(params)
+    np.random.seed(0)
+    particle.generate_particles(Np_tracer, seed_xloc, seed_yloc)
+    old_walk_data = particle.run_iteration()
+    # try to do another walk - test just makes sure code doesn't break
+    np.random.seed(0)
+    all_walk_data = particle.run_iteration()
+    assert all_walk_data['xinds'][0][0] == 1
+    assert all_walk_data['yinds'][0][0] == 1
+    assert all_walk_data['travel_times'][0][0] == 0.0
+
+
+def test_generate_twice():
+    # test ability to generate particles multiple times
+    particle = particle_track.Particles(params)
     np.random.seed(0)
+    old_init = particle.generate_particles(Np_tracer, seed_xloc, seed_yloc)
+    old_init = particle.generate_particles(Np_tracer, seed_xloc, seed_yloc,
+                                           previous_walk_data=old_init)
```

Comment:
Looks like you can just remove the mentions of `old_init`, should still do the same thing internally

Reply:
Thanks :+1: